dotcom-rendering/src/web/components/PullQuoteBlockComponent.stories.tsx

```diff
@@ -22,12 +71,8 @@ export const SportInline = () => {
 			showSideBorders={false}
 		>
 			<PullQuoteBlockComponent
-				design={ArticleDesign.Standard}
-				palette={decidePalette({
-					display: ArticleDisplay.Standard,
-					design: ArticleDesign.Standard,
-					theme: ArticlePillar.Sport,
-				})}
+				format={format.sport}
+				palette={decidePalette(format.sport)}
```

Comment:
Instead of putting all the formats together in this `format` object (which could be confusing from its naming), you could do something like

```tsx
const defaultFormat = {
    display: ArticleDisplay.Standard,
    design: ArticleDesign.Standard,
    theme: ArticleTheme.News,
}

...
// For each of the formats you want to test
export const OpinionInline = () => {
    const format = {
        ...defaultFormat,
        theme: ArticlePillar.Opinion
    }
     ...
    <PullQuoteBlockComponent
          format={format}
          palette={decidePalette(format)}
          ...
    >
    ....
}

Reply:
I like this! I made the change, thanks